client/my-sites/themes/theme-showcase.jsx

```diff
@@ -198,6 +204,18 @@ class ThemeShowcase extends React.Component {
 				.sort( ( a, b ) => a.order - b.order )
 		);
 
+		const SampleBanner = () => (
+			<ThemesBanner
+				title={ translate( 'Are you a small business?' ) }
+				description={ translate(
+					"We understand your time contraints, and that's why we created this easy-to-set-up Premium theme called Small Business."
+				) }
+				ctaLabel={ translate( 'Learn about this theme' ) }
+				ctaAction={ this.helloWorld }
```

Comment:
We might have some prior art in Calypso where we call props like these `buttonLabel`/`buttonAction`; or, maybe even better, just `actionLabel` and `action`.

Reply:
Oh, and if you're planning to simply point to a different page, ideally just add a string prop named `buttonHref` (or actionHref` -- well, `buttonHref` might be better here).

Reply:
In this case it could be either or, the button could either point to a page, or run some code, in this case I think it's best to use `actionLabel` and `action` ;-)